src/SprykerEco/Zed/AvalaraTaxShipment/Business/AvalaraTaxShipmentFacadeInterface.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\AvalaraTaxShipment\Business;
+
+use Generated\Shared\Transfer\AvalaraCreateTransactionRequestTransfer;
+use Generated\Shared\Transfer\AvalaraCreateTransactionResponseTransfer;
+use Generated\Shared\Transfer\CalculableObjectTransfer;
+
+interface AvalaraTaxShipmentFacadeInterface
+{
+    /**
+     * Specification:
+     * - Expands `AvalaraCreateTransactionRequestTransfer` with shipments.
+     * - Requires `CalculableObjectTransfer.items.shipment.method.avalaraTaxCode` and `CalculableObjectTransfer.priceMode` to be set.
```

Comment:
* Expects `CalculableObjectTransfer.items.shipment.method.avalaraTaxCode` to be set.

Reply:
Resolved.